astronomer/providers/dbt/cloud/hooks/dbt.py

```diff
@@ -0,0 +1,127 @@
+from functools import wraps
+from inspect import signature
+from typing import Any, Dict, List, Optional, Tuple, TypeVar, cast
+
+import aiohttp
+from aiohttp import ClientResponseError
+from airflow import AirflowException
+from airflow.hooks.base import BaseHook
+from airflow.models import Connection
+from asgiref.sync import sync_to_async
+
+from astronomer.providers.package import get_provider_info
+
+T = TypeVar("T", bound=Any)
+
+
+def provide_account_id(func: T) -> T:
+    """
+    Decorator which provides a fallback value for ``account_id``. If the ``account_id`` is None or not passed
+    to the decorated function, the value will be taken from the configured dbt Cloud Airflow Connection.
+    """
+    function_signature = signature(func)
+
+    @wraps(func)
+    async def wrapper(*args: Any, **kwargs: Any) -> Any:
+        bound_args = function_signature.bind(*args, **kwargs)
+
+        if bound_args.arguments.get("account_id") is None:
+            self = args[0]
+            if self.dbt_cloud_conn_id:
+                connection = await sync_to_async(self.get_connection)(self.dbt_cloud_conn_id)
+                print(connection)
```

Comment:
Is it necessary to print the connection here?

Reply:
No its necessary to print it I used it for my local development will remove it